### DISCUSSION TITLE
Tema claro e font-size maior

### DISCUSSION THREAD

**Discussion by dennis-rocha** [dgadelha/Portugol-Webstudio#202] [Ideias e sugestões]
Olá,

Uma sugestão muito agradável seria a possibilidade de trocar para o tema branco e aumentar o tamanho da font.
Uso o Portugol-Webstudio nas aulas e possuo dificuldades de projetar o App no telão pois o tema escuro não é visualmente eficiente quando projeta a tela. 

Inclusive quando projeto a tela eu aumento a fonte dando zoom na tela e isso faz com que todos os elementos da tela aumente também. Então seria interessante a possibilidade de possui um botão que pudesse aumentar ou diminuir o tamanho da font-size da IDE.

Seria muito interessante possuir um botão de configurações na lateral esquerda com as sugestões acima.

Estou disponível para aprender e cooperar no projeto :handshake:

**Comment by prcvilla** [comment]
Você pode trocar para o tema claro usando o console do navegador:
```js
monaco.editor.setTheme('vs')
```
E para aumentar a fonte:
```js
monaco.editor.EditorZoom.setZoomLevel(5) 
```

**Comment by dngadelha** [comment]
Olá, @dennis-rocha!

Adicionamos opções para alterar o tema da IDE para claro ou escuro e alterar o tamanho da fonte.

![image](https://github.com/user-attachments/assets/1cffc15b-7bb2-443c-b0c4-c048c06e016e)

Essas opções já estão disponíveis em https://portugol.dev/.
